ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1608,3 +1608,58 @@
                 default-excludes: true
                 fingerprint: false
                 only-if-success: false
+      - 'cloud-{name}-{test_name}':
+          disabled: false
+          triggers: []
+          test_name: test-aws-sriov-secondary-network
+          node: antrea-cloud
+          description: 'Test SR-IOV secondary network in AWS for antrea.'
+          branches:
+          - '${{ANTREA_GIT_REVISION}}'
+          repo_url: '${{ANTREA_REPO}}'
+          builders:
+          - shell: |-
+              #!/bin/bash
+              cat ${{AWS_EC2_SSH_KEY}} > ./${{AWS_EC2_SSH_KEY_NAME}}
+              chmod 400 ${{AWS_EC2_SSH_KEY_NAME}}
+              DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+              [ "$DOCKER_REGISTRY" != "docker.io" ] || ./ci/jenkins/docker_login.sh --docker-user ${{DOCKER_USERNAME}} --docker-password ${{DOCKER_PASSWORD}}
```

Comment:
Why to check `[ "$DOCKER_REGISTRY" != "docker.io" ] `, I think the user and password are for dockerhub and we should always do the login to avoid image pull rate limit. @XinShuYang  can you confirm?

Reply:
@luolanzone  Since we are encountering the rate limit issue with dockerhub images, we currently have to log in to a docker account to avoid it. However, if we can have a new repo without this limitation (similar to the Harbor repo we used before), we will no longer need to log in to the docker account.